spring-kafka/src/main/java/org/springframework/kafka/support/micrometer/DefaultKafkaListenerObservationConvention.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.kafka.support.micrometer;
+
+import io.micrometer.common.KeyValues;
+
+/**
+ * Default {@link KafkaListenerObservationConvention} for Kafka listener key values.
+ *
+ * @author Gary Russell
+ * @since 3.0
+ *
+ */
+public class DefaultKafkaListenerObservationConvention implements KafkaListenerObservationConvention {
+
+	/**
+	 * A singleton instance of the convention.
+	 */
+	public static final DefaultKafkaListenerObservationConvention INSTANCE =
+			new DefaultKafkaListenerObservationConvention();
+
+	@Override
+	public KeyValues getLowCardinalityKeyValues(KafkaRecordReceiverContext context) {
+		return KeyValues.of(KafkaListenerObservation.ListenerLowCardinalityTags.LISTENER_ID.asString(),
+						context.getListenerId());
+	}
+
+	@Override
+	public KeyValues getHighCardinalityKeyValues(KafkaRecordReceiverContext context) {
+		return KeyValues.empty();
```

Comment:
It looks like `default` method on the `ObservationConvention` does exactly the same